media-libs/openexr/openexr-2.2.0-r1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit eutils multilib-minimal
```

Comment:
No idea. Was just following the example on the wiki page. I figured I wouldn't chance anything.